test/preseed/provision_test.rb

```diff
@@ -21,10 +21,12 @@
     releases.each do |major, minor, release|
       describe "#{distro} #{major}.#{minor} #{release}" do
         it 'will compile' do
-          code, stdout, stderr = @renderer.render(distro, major, minor, release)
-          _(stdout).must_equal ''
-          _(stderr).must_equal ''
-          _(code).must_equal 0
+          skip("debconf-set-selections not found, use DEBCONFSETSEL to set path to executable") unless @renderer.debian_validator_present?
+          code, stdout, stderr, contents = @renderer.render(distro, major, minor, release)
+          puts contents if code != 0 || stdout != '' || stderr != ''
+          stdout.must_equal ''
```

Comment:
This looks like the older minitest syntax that generates deprecations in the older versions.

Reply:
Yeah I did not bother, let's fix it.